baseapp/config.go

```diff
@@ -27,3 +27,10 @@ type HTTPConfig struct {
 	PublicURL string     `yaml:"public_url" json:"publicUrl"`
 	TLSConfig *TLSConfig `yaml:"tls_config" json:"tlsConfig"`
 }
+
+// LoggingConfig contains options for logging, such as log level and textual representation.
+// It is usually embedded in a larger configuration struct.
+type LoggingConfig struct {
+	Level string `yaml:"level"`
+	Text  bool   `yaml:"text"`
```

Comment:
I'd like to add a comment here, or maybe change the field name: JSON is also a text format, so I want to clarify that setting this option makes output human-readable / pretty.